src/options.js

```diff
@@ -45,211 +46,214 @@ export default {
-      redirectUri: window.location.origin + '/',
-      requiredUrlParams: ['display', 'scope'],
-      scope: ['email'],
-      scopeDelimiter: ',',
-      display: 'popup',
-      oauthType: '2.0',
-      popupOptions: { width: 580, height: 400 }
+      name: "facebook",
+      url: "/auth/facebook",
+      authorizationEndpoint: "https://www.facebook.com/v2.11/dialog/oauth",
+      redirectUri: window.location.origin + "/",
+      requiredUrlParams: ["display", "scope"],
+      scope: ["email"],
+      scopeDelimiter: ",",
+      display: "popup",
+      oauthType: "2.0",
+      popupOptions: { width: 580, height: 400 },
     },
 
     google: {
-      name: 'google',
-      url: '/auth/google',
-      authorizationEndpoint: 'https://accounts.google.com/o/oauth2/auth',
+      name: "google",
+      url: "/auth/google",
+      authorizationEndpoint: "https://accounts.google.com/o/oauth2/auth",
       redirectUri: window.location.origin,
-      requiredUrlParams: ['scope'],
-      optionalUrlParams: ['display'],
-      scope: ['profile', 'email'],
-      scopePrefix: 'openid',
-      scopeDelimiter: ' ',
-      display: 'popup',
-      oauthType: '2.0',
-      popupOptions: { width: 452, height: 633 }
+      requiredUrlParams: ["scope"],
+      optionalUrlParams: ["display"],
+      scope: ["profile", "email"],
+      scopePrefix: "openid",
+      scopeDelimiter: " ",
+      display: "popup",
+      oauthType: "2.0",
+      popupOptions: { width: 452, height: 633 },
     },
 
     github: {
-      name: 'github',
-      url: '/auth/github',
-      authorizationEndpoint: 'https://github.com/login/oauth/authorize',
+      name: "github",
+      url: "/auth/github",
+      authorizationEndpoint: "https://github.com/login/oauth/authorize",
       redirectUri: window.location.origin,
-      optionalUrlParams: ['scope'],
-      scope: ['user:email'],
-      scopeDelimiter: ' ',
-      oauthType: '2.0',
-      popupOptions: { width: 1020, height: 618 }
+      optionalUrlParams: ["scope"],
+      scope: ["user:email"],
+      scopeDelimiter: " ",
+      oauthType: "2.0",
+      popupOptions: { width: 1020, height: 618 },
     },
 
     instagram: {
-      name: 'instagram',
-      url: '/auth/instagram',
-      authorizationEndpoint: 'https://api.instagram.com/oauth/authorize',
+      name: "instagram",
+      url: "/auth/instagram",
+      authorizationEndpoint: "https://api.instagram.com/oauth/authorize",
       redirectUri: window.location.origin,
-      requiredUrlParams: ['scope'],
-      scope: ['basic'],
-      scopeDelimiter: '+',
-      oauthType: '2.0',
-      popupOptions: { width: null, height: null }
+      requiredUrlParams: ["scope"],
+      scope: ["basic"],
+      scopeDelimiter: "+",
+      oauthType: "2.0",
+      popupOptions: { width: null, height: null },
     },
 
     twitter: {
-      name: 'twitter',
-      url: '/auth/twitter',
-      authorizationEndpoint: 'https://api.twitter.com/oauth/authenticate',
```

Comment:
@onegreypaw This is it here. Changing the endpoint to /authorize from /authenticate.

Reply:
Are we saying with this change everything else on the playr/playr-api side stays the same as it was (with oauth 1)? Or are all those changes still needed?

Reply:
Those changes aren't needed, looks like whatever changes they made only apply to the previous endpoint. It feels like we are still at the mercy of Twitter, who's to say they don't apply the changes to this endpoint as they did to the previous endpoint